DOCS-2055: update bancontact postepay visa vpay testing details (#1805)
MultiSafepay/docs: content/payments/methods/wallet/paypal/integration-and-testing.md

---

**Note**: 

> - You can't test sending a redirect API request.

Reviewer: ```suggestion
- You can't test sending redirect API requests.
```

> - In the live environment it is not possible to accept a refund order. This is done automatically.

Reviewer: ```suggestion
- In the live environment, you can't accept refund orders. These are done automatically.
```
